src/Selenium2Library/librarycomponent.py

```diff
@@ -2,15 +2,16 @@
 from selenium.webdriver.remote.webelement import WebElement
 
 from .locators.elementfinder import ElementFinder
+from Selenium2Library.contextaware import ContextAware
```

Comment:
Somewhat inconsistent to use `.module` with one import and `Selenium2Library.module` with the other.